scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf

```diff
@@ -1,12 +1,9 @@
-Node: E:LocalGovernmentDirectory_Districts->E0
-typeOf: dcs:Place
-lgdCode: C:LocalGovernmentDirectory_Districts->LGDStateCode
-
 Node: E:LocalGovernmentDirectory_Districts->E1
 typeOf: dcs:AdministrativeArea2
+dcid: C:LocalGovernmentDirectory_Districts->DistrictDCID 
 wikidataId: C:LocalGovernmentDirectory_Districts->WikiDataId
 name: C:LocalGovernmentDirectory_Districts->LGDDistrictNameTitleCase
 lgdCode: C:LocalGovernmentDirectory_Districts->LGDDistrictCode
 indianCensusAreaCode2011: C:LocalGovernmentDirectory_Districts->LGDCensus2011Code
 indianCensusAreaCode2001:  C:LocalGovernmentDirectory_Districts->LGDCensus2001Code
-containedInPlace: E:LocalGovernmentDirectory_Districts->E0
+containedInPlace: E:LocalGovernmentDirectory_Districts->StateDCID
```

Comment:
Please change to column reference form node referes with `C:` prefix:
`containedInPlace: C:LocalGovernmentDirectory_Districts->StateDCID`

Reply:
Fixed